dgraphql/schema/bindata.go

```diff
@@ -1,22 +1,22 @@
-// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
Seems there is a mismatch between the version of `bindata` we use and the one you used, the format changed quite a bit here.

I don't think it has an incidence thought on the functionality.  

What do you have for:

```
$ go-bindata --version
go-bindata 3.22.0 (Go runtime go1.15.3).
Copyright (c) 2010-2015, Jim Teeuwen.
Copyright (c) 2017-2020, Kevin Burke.
```
?

Reply:
Was running an older version of go-bindata. Updated to 3.22 and re-generated the file.